app/pages/home.spec.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import assert from 'assert';
+import { shallow } from 'enzyme';
+import HomePageRoot from './home';
+
+const user = {
+  display_name: 'Test User'
+}
+
+const location = {
+  hash: '#page'
+}
+
+describe('HomePageRoot', function () {
+  describe('renders a homepage', function() {
+    let wrapper;
+
+    beforeEach(function () {
+      wrapper = shallow(<HomePageRoot user={user} location={location} />)
+    });
+
+    it('without crashing', function () {
+      assert.equal(wrapper, wrapper)
+    });
+
+    it('with a user prop', function() {
```

Comment:
This is testing the child component, not really `HomePageRoot`. This test would be more appropriate for testing `HomePageLoggedIn`.